src/main/java/org/folio/rest/impl/TenantRulesImpl.java

```diff
@@ -178,4 +181,16 @@ private String validateRule(Rule entity) {
     return errorMessage;
   }
 
+  private String validateRuleOnPut(Rule entity) {
+    if (StringUtils.isEmpty(entity.getId()) || StringUtils.isEmpty(entity.getRuleId())) {
+      logger.debug("Rule id cannot be null or empty");
+      return RULE_ID_EMPTY_ERROR;
+    } else if (!StringUtils.isEmpty(entity.getId()) && !StringUtils.isEmpty(entity.getRuleId())
+      && !entity.getId().equals(entity.getRuleId())) {
```

Comment:
These two lines can be reduced to

    }
    if (! entity.getId().equals(entity.getRuleId())) {

Reply:
Replaced, thank you